app/scripts/services/utils.js

```diff
@@ -52,12 +52,13 @@ angular.module('oncokbStaticApp')
                     'NA') : 'NA';
             },
             getVariantCellContent: function(geneName, variantName) {
-                if (geneName && variantName && variantName.indexOf('/') === -1) {
-                    geneName = geneName.trim();
-                    variantName = variantName.trim();
-                    return '<a href="#/gene/' + geneName + '/variant/' + variantName + '">' + variantName + '</a>';
+                geneName = geneName.trim();
+                variantName = variantName.trim();
+                var formattedVariantName = variantName;
+                if (/\//.test(variantName)) {
+                    formattedVariantName = encodeURIComponent(encodeURIComponent(formattedVariantName));
```

Comment:
If you encode it only once, the server decodes it, then it's the same as not encoding it at all.